lib/src/golden_test_adapter.dart

```diff
@@ -376,13 +377,70 @@ class FlutterGoldenTestWrapper extends StatelessWidget {
     return MediaQuery(
       data: MediaQuery.maybeOf(context) ??
           MediaQueryData.fromWindow(WidgetsBinding.instance.window),
-      child: Directionality(
-        textDirection: Directionality.maybeOf(context) ?? TextDirection.ltr,
```

Comment:
don't we need any directionality anymore?

Reply:
It's provided by the `Localizations` widget as well.